twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/cassandra.ts

```diff
@@ -164,6 +160,11 @@ export class CassandraConnector extends AbstractConnector<
       return this;
     }
 
+    if (this.options.elasticsearch) {
```

Comment:
Is it up to the ORM to manage elasticsearch?

Here is what I was thinking about the search: Create a `@Index` decorator and put it everywhere we want to index resources. The search is a separated core service which has nothing to deal with ORM.

Reply:
I first though like you, then I though that I want mongodb installs to work without elasticsearch (using regex calls). This is why I decided to move it in the ORM in a way to "extends" the scylladb capabilities.
I created an extension of the decorator @entity decorator, to choose what to index or not at the upsert time.
We can address this tomorrow in daily?

Reply:
I will move it to platform service